org.freeciv.gtk4.yml

```diff
@@ -0,0 +1,28 @@
+app-id: org.freeciv.gtk4
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+command: freeciv-gtk4
+rename-icon: freeciv-client
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=~/.freeciv:create
```

Comment:
Why aren't you using `persist=.freeciv` here?

Reply:
I've understood that 'persist' creates location specific for that flatpak, so wouldn't work for the goal of sharing data between freeciv.org flatpaks.

Reply:
Yea sharing configs with other apps won't work.